src/index.html

```diff
@@ -9,6 +9,44 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="logo">
+        <img 
+          src="images/logo.png"
+          alt="MOYO logo" 
+          class="moyo-logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link apple is-active">APPLE</a>
+          </li>
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link samsung">SAMSUNG</a>
```

Comment:
Don't use uppercase letters, let's find out how to do it in the CSS
```suggestion
            <a href="#samsung" class="nav__link samsung">SAMSUNG</a>
```